core/modules/server/routes/get-tiddler-html.js

```diff
@@ -18,12 +18,12 @@ exports.path = /^\/view\/([^\/]+)$/;
 
 exports.handler = function(request,response,state) {
 	var title = decodeURIComponent(state.params[0]),
-		tiddler = state.wiki.getTiddler(title);
+	    tiddler = state.wiki.getTiddler(title);
```

Comment:
TW uses tabs as whitespace. tabs and spaces should __not__ be mixed!

Reply:
My apologies, I thought I fixed that earlier, should be cleaned up now.